public/update_team.php

```diff
@@ -0,0 +1,159 @@
+        }
+        
+    } catch (PDOException $error) {
+        echo "<br>" . $error->getMessage();
+    }
+}
+
+$available_students = array();
+foreach ($course_students as $student) {
+    if (!in_array($student->id, $students_in_teams)){
+        array_push($available_students, $student);
+    }
+    
+}
+
+if (isset($_POST["submit"])) {
+
+    $team_to_edit_id = $_POST['team'];
+    $students_to_remove = $_POST['team_member_delete'] ?? [];
+    
+
+    
+    foreach($students_to_remove as $remove){
+        $delete = TeamMember::find_by(array('user_id' => $remove, 'team_id' => $team_to_edit_id ));
+        if (!is_null($delete)){
+            $delete->delete();
+        }
+  
+    }
+    $number_team_members = TeamMember::where(array('team_id' => $team_to_edit_id));
+    if(count($number_team_members)==0){
+        $delete_team = Team::find_by(array('id' => $team_to_edit_id));
+        $delete_team->delete();
+    }
+    
+
+    if($_POST['student_selection'] != null){
+        $team_member = new TeamMember();
+        $team_member->team_id = $team_to_edit_id;
+        $team_member->user_id = $_POST['student_selection'];
+
+        try {
+            $team_member->save();
+        } catch (PDOException $error) {
+            echo "<br>" . $error->getMessage();
+        }
+    }
+    echo "<h5>Team updated succesfully </h5>";
+    header("refresh : 1");
+    
+}
+
+?>
+
+<h2>Teams</h2>
+    <?php if ($teams && count($teams)) { ?>
+    
+        <?php foreach ($teams as $row) { ?>
+           <h3><?php echo "Team # ".$row->id ?></h3>
+           <form method="post">
+           <table>
+                <thead>
+                    <tr>
+                        <th>#</th>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>Email Address</th>
+                        <th>Created At</th>
+                    </tr>
+                </thead>
+                <tbody>
+                <?php $students_in_team = $row->team_member; 
+                foreach ($students_in_team as $student) {?>
+                <tr>
+                    <td><?php echo escape($student->user_id); ?></td>
+                    <td><?php echo escape($student->user->first_name); ?></td>
+                    <td><?php echo escape($student->user->last_name); ?></td>
+                    <td><?php echo escape($student->user->email); ?></td>
+                    <td><?php echo escape($student->user->created_at);  ?> </td>
+                    <td><input class="single-checkbox" type="checkbox" id="<?php echo $student->user_id; ?>" name="team_member_delete[]" value="<?php echo $student->user_id; ?>"> Remove from team</td>
+                <?php }?>
+                </tbody>
+            </table>
+            <?php if (count($students_in_team)<4){ ?>
+                Add Student: 
+                <select Name="student_selection" id="student_selection">
+                    <option value="">----Select----</option>
```

Comment:
I stand by my point we should make it `disabled selected`

Reply:
no I disagree, because then if you make a mistake and selecte a student to be added and dont want to add a student you are stuck

Reply:
ok I take back what I said, I just realised what youre doing there.